app/views/items/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<html>
+  <head>
+    <%= stylesheet_link_tag "main" %>
+  </head>
+
+  <body>
+    <div class="pageheaders">
+      <h1>All Items</h1>
+    </div>
+
+    <div class="itemstables">
+
+    <table id="t01">
+      <tr>
+        <th>Name</th>
+        <th>Description</th>
+        <th>Image</th>
+        <th>Price</th>
+        <th>Status</th>
+        <th>Inventory</th>
+        <th>Merchant</th>
+      </tr>
+      <% @items.each do |item| %>
+      <tr>
+        <td><a style="color: #000000; text-decoration: underline;" href= <%= "/items/#{item.id}"%>> <%= item.name %></a></td>
+        <td><%= item.description %></td>
+        <td><img src = <%= item.image %> ></img></td>
+        <td><%= number_to_currency(item.price) %></td>
+        <td><%= item.status %></td>
+        <td><%= item.inventory %></td>
+        <td><a style="color: #000000; text-decoration: underline;" href= <%= "/merchants/#{item.merchant_id}"%>> <%= item.merchant.name %> </a></td>
+      </tr>
```

Comment:
All of this above looks good, but since we're in rails, we want you to try to emphasize using rails convention, and part of that convention is using action view helpers when they're available. In rails, link_to is generating an anchor tag for you, so instead of <a> use a link_to in erb. Similarly with img, you could use image_tag. I fall in and out of this rails convention myself, but it's just something to be aware of and to try and make yourself more conscious of while creating your views.